src/segment-loader.js

```diff
@@ -1104,4 +1097,12 @@ export default class SegmentLoader extends videojs.EventTarget {
     this.throughput.rate +=
       (segmentProcessingThroughput - rate) / (++this.throughput.count);
   }
+
+  /**
+   * A debugging logger noop that is set to console.log only if debugging
+   * is enabled globally
+   *
+   * @private
+   */
+  logger_() {}
```

Comment:
I plan on doing this later by pulling in Pat's logger/history thing. This change was just to get rid of that ugly hack since we already have support for a less-ugly option in playback-watcher.